src/main/java/gov/usgs/owi/nldi/controllers/HtmlController.java

```diff
@@ -25,35 +26,65 @@
 	@GetMapping(value="/linked-data/{featureSource}/**", produces= MediaType.TEXT_HTML_VALUE)
 	public String getLinkedDataHtml(HttpServletRequest request, HttpServletResponse response,
 		@RequestParam(name= Parameters.FORMAT, required=false) @Pattern(regexp=BaseController.OUTPUT_FORMAT) String format) throws Exception {
-		
+		return processHtml(request, response);
+	}
+
+	@GetMapping(value="/linked-data", produces= MediaType.TEXT_HTML_VALUE)
+	public String getLinkedDataDataSourcesHtml(
+		HttpServletRequest request, HttpServletResponse response,
+		@RequestParam(name= Parameters.FORMAT, required=false) @Pattern(regexp=BaseController.OUTPUT_FORMAT) String format) throws Exception {
 		return processHtml(request, response);
 	}
 
 	@GetMapping(value="/linked-data/comid/**", produces= MediaType.TEXT_HTML_VALUE)
 	public String getNetworkHtml(HttpServletRequest request, HttpServletResponse response,
 		@RequestParam(name=Parameters.FORMAT, required=false) @Pattern(regexp=BaseController.OUTPUT_FORMAT) String format) throws Exception {
-		
 		return processHtml(request, response);
 	}
 
 	@GetMapping(value="/lookups/**", produces= MediaType.TEXT_HTML_VALUE)
 	public String getLookupsHtml(HttpServletRequest request, HttpServletResponse response,
 		@RequestParam(name=Parameters.FORMAT, required=false) @Pattern(regexp=BaseController.OUTPUT_FORMAT) String format) throws Exception {
-		
 		return processHtml(request, response);
 	}
 
-	private String getHtmlString(HttpServletRequest request) throws IOException {
+	private String removeHtmlFormatFromQueryString(String oldQueryString) {
```

Comment:
This a fix for the HTML (PR #161).  

When the user is in the browser and lands on an HTML page, it needs to display a link to the same page but in JSON mode (by using the same URL, but f=json instead of f=html).  This removes f=html if it exists in the query string.